maat-orchestration/build.gradle

```diff
@@ -19,7 +19,7 @@ def versions = [
 		pitest                  : "1.16.1",
 		springdocVersion        : "2.5.0",
 		commonsModSchemas       : "1.5.0",
-		crimeCommonsClasses     : "3.22.0",
+		crimeCommonsClasses     : "3.29.0-SNAPSHOT",
```

Comment:
Just wondering, should this be a snapshot or should we be using a non-snapshot version of crime-commons?

Reply:
Ignore this, I was going through commit by commit and didn't realise you had changed this in a later commit.

Reply:
No worries!